### DISCUSSION TITLE
How to use custom day labels in v-calendar?

### DISCUSSION THREAD

**Discussion by thany** [vuetifyjs/vuetify#19884] [Q&A]
This was working in v2:

```
<template v-slot:day-label-header="{ date }">
  // Do something with date or whatever and poop out a <v-btn>
</template>
```

Not anymore in v3. So how do I translate this to vuetify 3?

I did see something in the documentation about "exposed" and an entry `genDays`, but what is meant with exposed? Exposed how & where? And what can we do with exposed things in this regard? There is no documentation on what "exposed" means, or even so much as an example.

Maybe `genDays` isn't even what I need. It doesn't matter what I need - I just need to achieve my goal, and that is to apply custom styling (could be a class or whatever, I don't care as long as I can apply *some* sort of custom style) to a day header based on business logic. Currently I haven't got a clue how to get there, and so anything would be better than what I currently have, which is nothing.

**Comment by blalan05** [comment]
This feature has not been added back yet. There is a PR open that includes many of the slots and click events, but needs slightly more polish and then approved.
